Makefile

```diff
@@ -326,6 +326,14 @@ static-checks: vendor
 fix:
 	goimports -w $(SRCFILES)
 
+foss-checks: vendor
```

Comment:
Should this be called by the ci target?

Reply:
Good question.  I think it depends on whether we expect people to run `make ci` on their machines or if that target is only used on Semaphore.

If the `ci` target is only supposed to be used on Semaphore, then yes, I agree that I need to add `foss-checks` to the `ci` target.

@caseydavenport - care to make a call?

Reply:
I think the plan here is that we don't call this from the CI target and instead explicitly call from Semaphore on the master branch only.